auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/UmsClient.java

```diff
@@ -654,4 +655,34 @@ public String getIdentityProviderMetadataXml(String requestId, String accountId)
                 .build());
     }
 
+    public UserManagementProto.SetWorkloadAdministrationGroupNameResponse setWorkloadAdministrationGroupName(String requestId, String accountId,
+            String right, String resource) {
+        checkNotNull(accountId);
```

Comment:
Are right and resource required for these requests? If so then we should null-check them also.